### PR TITLE
Fix some lastsplit options

### PR DESCRIPTION
**Pull request by RomainDallet**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by bgruening — COMMENTED** [review]
Hi @RomainDallet,

can you explain why this is needed? 

Thanks,
Bjoern

**Comment by RomainDallet** [comment]
> Hi @RomainDallet,
> 
> can you explain why this is needed?
> 
> Thanks,
> Bjoern
Hi @bgruening 

The default values ​​set was completely arbitrary, depending on the dataset I was using. I hadn't seen that they were still in the wrapper.
The tool documentation does not indicate default values, so I prefer to put them optional.

Romain

File: tools/last/lastsplit.xml [hunk 1]
```diff
@@ -25,11 +25,23 @@
         #end if
 
         -d $lastsplit_opt.d
-        -c $lastsplit_opt.c
-        -t $lastsplit_opt.t
-        -M $lastsplit_opt.M
-        -S $lastsplit_opt.S
-        -m $lastsplit_opt.m
+
+        #if $lastsplit_opt.c != ""
```
Comment (bgruening) [inline tools/last/lastsplit.xml:29]: @RomainDallet can you replace this by `#if str($lastsplit_opt.c):`

**Review by bernt-matthias — COMMENTED** [review]
Seems that the defaults have been chosen in a useful way. Check the sources for `last-split-main.cc`:

```
static void run(int argc, char* argv[]) {
  LastSplitOptions opts;

  opts.format = 'M';
  opts.direction = 1;
  opts.cis = 0.004;
  opts.trans = 1e-05;
  opts.mean = 7.0;
  opts.sdev = 1.7;
  opts.mismap = 1.0;
  opts.score = -1;
  opts.no_split = false;
  opts.bytes = 0;
  opts.verbose = false;
  opts.isSplicedAlignment = false;
```

So please revert, or mention the defaults in the help.

**Comment by RomainDallet** [comment]
Indeed, I had not found this file (last-split-main.cc). So the changes are unnecessary. Thanks for the review!

**Pull request closed without merging**